### PR TITLE
Close the previous image file before creating a new one

### PR DESCRIPTION
**Pull request by rmdlv**

### CODE REVIEW

**Comment by rmdlv** [comment]
Oops... VSCode formatted the whole project... But maybe that's even better?

My changes only affect the following lines:: https://github.com/baskiton/GeoscanDecoder/pull/3/commits/5fd47fd086066afcfb97afecd12bae3cf3ecb01a#diff-c3f3410608bda4ec080e6199be70c0f81a909d4f3623cf2eec4c40127e236fc6R157-R159

File: GeoscanDecoder/geoscan.py [hunk 1]
```diff
@@ -149,16 +148,21 @@ def set_merge_mode(self, val):
...
-            self.current_fid = f'GEOSCAN_{dt.datetime.now()}'.replace(' ', '_').replace(':', '-')
+            self.current_fid = f"GEOSCAN_{dt.datetime.now()}".replace(" ", "_").replace(
+                ":", "-"
+            )
         return self.current_fid
 
     def force_new(self):
+        f = self.files.get(self.current_fid)
+        if f:
+            f.close()
```
Comment (baskiton) [inline GeoscanDecoder/geoscan.py:158]: Good!

**Review by baskiton — CHANGES_REQUESTED** [review]
no, that's not better.
First, the codestyle is broken.
Secondly, history breaks down.
third, change for the sake of change.
fourthly, in a pull request, all changes should relate only to what is described.

File: GeoscanDecoder/geoscan.py (hunk 1 above)
Comment (rmdlv) [reply GeoscanDecoder/geoscan.py:158]: Ok, but I had to create a .vscode directory in the project and add it to .gitignore to disable autoformatting.

**Review by baskiton — APPROVED** [review]

**Pull request merged**